arraycontext/container/traversal.py

```diff
@@ -533,6 +533,9 @@ def freeze(
 
     See :meth:`ArrayContext.thaw`.
     """
+    if actx is not None and actx.supports_freezing_containers:
```

Comment:
`actx.thaw` should then also support containers, for symmetry.

Reply:
Given that we are not replacing `freeze(..., actx)` with `actx.freeze`, I don't think we would be quite preserving symmetry here.

Reply:
Fair enough.

Reply:
It looks like this has already happened for pytato and pyopencl, but not for Jax. Make it consistent?

Reply:
EagerJax and PytatoJax already support `actx.(freeze|thaw)` on containers. Anything I'm missing?

Reply:
Sorry, misread this code. You're right!

Reply:
Great! In that case, ok to orchestrate the merges of this andhttps://github.com/inducer/meshmode/pull/327?

Reply:
(Already on the way!)